daemon/config/config.go

```diff
@@ -53,7 +54,11 @@ const (
 	DefaultContainersNamespace = "moby"
 	// DefaultPluginNamespace is the name of the default containerd namespace used for plugins.
 	DefaultPluginNamespace = "plugins.moby"
-
+	// DefaultMinAPIVersion is the minimum API version supported by the API.
+	// This version can be overridden through the "DOCKER_MIN_API_VERSION"
+	// environment variable. The minimum allowed version is determined
+	// by [minAPIVersion].
+	DefaultMinAPIVersion = "1.24"
```

Comment:
I think this constant should also be unexported. Daemon code outside of this package has no legitimate need to reference this constant, and doubly so for external modules.

Reply:
ah, you're right; don't think we use it outside of the package now, so can do when I'm back at my computer

Reply:
rebased, and un-exported the const